### PR TITLE
fix: update generator-client/README.md

### PR DESCRIPTION
**Pull request by stevanlohja**
This PR is an update for the root [generator-client/`README.md`] only

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,71 @@
 # open-rpc-generator-client
-Multi-language client generators for [open-rpc](https://github.com/open-rpc/spec) apis.
 
-[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=open-rpc/generator-client)](https://dependabot.com)
+[![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=open-rpc/generator-client)](https://dependabot.com) [![Dependabot Status](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/Tf55C7d)
 
-Install using npm
-```
+Multi-language client generator tool for [open-rpc](https://github.com/open-rpc/spec) APIs. Developers generate a client by passing a raw open-rpc schema document to _this_ tool which generates a client based on the supported language templates.
```
Comment (shanejonas) [inline README.md]:
```suggestion
Multi-language client generator tool for [open-rpc](https://github.com/open-rpc/spec) APIs. Developers generate a client by passing an OpenRPC Document to _this_ tool which generates a client based on the supported language templates.
```

File: README.md [hunk 2]
```diff
@@ -1,33 +1,71 @@
...
+
 In action (for a js client):
-```sh
+
+```shell
 docker run \
   -v $(pwd)/petstore/:/petstore \
-  openrpc/generator-client \
-    -s https://raw.githubusercontent.com/open-rpc/examples/master/service-descriptions/petstore-openrpc.json
+  openrpc/generator-client-js \
```
Comment (shanejonas) [inline README.md]:
```suggestion
  openrpc/generator-client \
```

File: README.md [hunk 3]
```diff
@@ -1,33 +1,71 @@
...
+
+### Generating a Client
+
+```shell
 open-rpc-generator-client \
   -s https://raw.githubusercontent.com/open-rpc/examples/master/service-descriptions/petstore-openrpc.json \
   PetStore
 ```
 
+Using the `open-rpc-generator-client` command, then passing an example schema document `petstore-openrpc.json` in the directory of `Petstore`.
```
Comment (shanejonas) [inline README.md]:
```suggestion
Using the `open-rpc-generator-client` command, then passing an example OpenRPC Document `petstore-openrpc.json` in the directory of `Petstore`.
```

**Comment by stevanlohja** [comment]
> fix up `OpenRPC Document` naming

commit your suggestions.

**Pull request merged**